components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/utils/message/PulsarMessageHeaders.java

```diff
@@ -28,4 +28,7 @@
     String KEY_BYTES = "key_bytes";
     String TOPIC_NAME = "topic_name";
     String MESSAGE_RECEIPT = "message_receipt";
+    String KEY_OUT = "CamelPulsarProducerMessageKey";
```

Comment:
I should note here that this header format seems to be more standard throughout Camel, but I am reticent to change the above for backwards-compatibility reasons.